src/commands/scanner/run.ts

```diff
@@ -180,6 +196,9 @@ export default class Run extends ScannerCommand {
 			this.ux.log(messages.getMessage('output.filtersIgnoredCustom', []));
 		}
 
+		// if severty-for-error flag is used, we want to make sure the severities are normalized
+		if (this.flags['severity-for-error']) this.flags['normalize-severity'] = true;
```

Comment:
Nitpick: In general, we're using bracket syntax for ifs. So I'd say use that here.